StandaloneMmPkg/Core/FwVol.c

```diff
@@ -233,3 +237,81 @@ MmCoreFfsFindMmDriver (
 
   return Status;
 }
+
+/**
+  Dispatch all Standalone BFVs.
+  The BFVs will be shadowed into MMRAM, caller is responsible for calling
+  MmFreeShadowedBfv() to free the shadowed BFVs.
+
+**/
+VOID
+MmDispatchAllBfv (
+  VOID
+  )
+{
+  UINTN                       Index;
+  EFI_PEI_HOB_POINTERS        BfvHob;
+  EFI_FIRMWARE_VOLUME_HEADER  *Bfv;
+
+  mBfvList = AllocateZeroPool (sizeof (EFI_FIRMWARE_VOLUME_HEADER *) * MAX_MM_BFV_COUNT);
+  if (mBfvList == NULL) {

```

Comment:
Since you used hardcode macro, AllocatePool() is not necessary.
You can just define mMmFv[MAX_MM_FV_COUNT]